requirements.txt

```diff
@@ -1,7 +1,7 @@
 Pillow
 gcsfs
 luxonis-ml[data,nn_archive]==0.8.0
-onnx==1.17.0
+onnx>=1.17.0,<1.19.0
```

Comment:
Do we still need this with the `mapping` fix above?

Reply:
I don't think there's a reason we don't want `1.18.0`. If it works we should use the latest version possible